packages/enhanced-img/package.json

```
@@ -45,9 +45,13 @@
 		"@types/node": "^18.19.3",
 		"estree-walker": "^3.0.3",
 		"rollup": "^4.14.2",
-		"svelte": "^4.2.10",
+		"svelte": "^4.2.14",
```

we should keep this in sync with the other packages. no need to change it
```suggestion
		"svelte": "^4.2.10",
```

---

Ah, right. That was an oversight. This should be addressed along with the other review comment. 👌

---

> This should be addressed along with the other review comment. 

Thanks for addressing this. The other review comment was not addressed though